va/http.go

```diff
@@ -434,6 +433,12 @@ func (va *ValidationAuthorityImpl) processHTTPValidation(
 		return nil, nil, err
 	}
 
+	// newIPError wraps an error and the IP of the remote host in an IPError so
+	// we can display the IP in the problem details returned to the client.
+	newIPError := func(target *httpValidationTarget, err error) ipError {
```

Comment:
This should return `error` instead of `ipError`. In general it's risky to return a concrete error type instead of the error interface, due to the "boxed nil" problem (https://www.youtube.com/watch?v=ynoY2xz-F8s).

Reply:
Ah, yeah! Thanks.